plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaPlugin.java

```diff
@@ -43,4 +50,73 @@ public void testSpinup()
         assertNotNull(connector);
         connector.shutdown();
     }
+
+    @Test
+    public void testSslSpinup()
+            throws IOException
+    {
+        KafkaPlugin plugin = new KafkaPlugin();
+
+        ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
+        assertInstanceOf(factory, KafkaConnectorFactory.class);
+
+        String secret = "confluent";
+        Path keystorePath = Files.createTempFile("keystore", ".jks");
+        Path truststorePath = Files.createTempFile("truststore", ".jks");
+
+        createTempFile(keystorePath, secret);
+        createTempFile(truststorePath, secret);
+
+        Connector connector = factory.create(
+                "test-connector",
+                ImmutableMap.<String, String>builder()
+                        .put("kafka.table-names", "test")
+                        .put("kafka.nodes", "localhost:9092")
+                        .put("kafka.security-protocol", "SSL")
+                        .put("kafka.ssl.keystore.type", "JKS")
+                        .put("kafka.ssl.keystore.location", keystorePath.toString())
+                        .put("kafka.ssl.keystore.password", "keystore-password")
+                        .put("kafka.ssl.key.password", "key-password")
+                        .put("kafka.ssl.truststore.type", "JKS")
+                        .put("kafka.ssl.truststore.location", truststorePath.toString())
+                        .put("kafka.ssl.truststore.password", "truststore-password")
+                        .put("kafka.ssl.endpoint-identification-algorithm", "https")
+                        .build(),
+                new TestingConnectorContext());
+        assertNotNull(connector);
+        connector.shutdown();
+    }
+
+    @Test(expectedExceptions = ApplicationConfigurationException.class)
+    public void testSslSpindown()
+    {
+        KafkaPlugin plugin = new KafkaPlugin();
+
+        ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
+        assertInstanceOf(factory, KafkaConnectorFactory.class);
+        factory.create(
+                "test-connector",
+                ImmutableMap.<String, String>builder()
+                        .put("kafka.table-names", "test")
+                        .put("kafka.nodes", "localhost:9092")
+                        .put("kafka.security-protocol", "SSL")
+                        .put("kafka.ssl.keystore.type", "JKS")
+                        .put("kafka.ssl.keystore.location", "/not/a/real/path")
+                        .put("kafka.ssl.keystore.password", "keystore-password")
+                        .put("kafka.ssl.key.password", "key-password")
+                        .put("kafka.ssl.truststore.type", "JKS")
+                        .put("kafka.ssl.truststore.location", "/not/a/real/path")
```

Comment:
please have two tests (using `@DataProvider`?) where you can test each file property individually

Reply:
I added 2 tests w/o `DataProvider` if that's fine with you.